.github/workflows/sec-scan.yml

```diff
@@ -0,0 +1,60 @@
+##
+# This action runs trivy container scanning and anchore grype vulnerability
+# scanner for docker images and packages.
+##
+
+name: sec-scan
+
+on:
+  repository_dispatch:
+    types: [sec-scan-trigger]
```

Comment:
This should also be called more descriptively

Reply:
Changed it to `trivy-scan-dispatch`